include/hip/hcc_detail/driver_types.h

```diff
@@ -254,14 +254,14 @@ typedef struct hipMemcpy3DParms {
     hipArray_t srcArray;
     struct hipPos srcPos;
     struct hipPitchedPtr srcPtr;
-
     hipArray_t dstArray;
     struct hipPos dstPos;
     struct hipPitchedPtr dstPtr;
-
     struct hipExtent extent;
     enum hipMemcpyKind kind;
+} hipMemcpy3DParms;
 
+typedef struct HIP_MEMCPY3D {
```

Comment:
Thank's for splitting RT's and Driver's structs - it is a headache supporting it in hipify-clang. Are you going to use HIP_MEMCPY3D struct further in Driver's related HIP code?